src/Microsoft.Identity.Abstractions/DownstreamApi/IDownstreamApi.cs

```diff
@@ -236,5 +237,136 @@ public partial interface IDownstreamApi
             Action<DownstreamApiOptions>? downstreamApiOptionsOverride = null,
```

Comment:
The above has

```
#if NET6_0_OR_GREATER
        [RequiresUnreferencedCode("This method's implementations also use generic types and are not trim-friendly.")]
#endif
```

However no one was building for net6+ before. In order to make this attribute be compiled, you will need at least one TFM that is `net6.0` or greater.